mobile/examples/phi-3/android/app/build.gradle.kts

```diff
@@ -14,15 +14,10 @@ android {
         versionName = "1.0"
 
         testInstrumentationRunner = "androidx.test.runner.AndroidJUnitRunner"
-        externalNativeBuild {
-            cmake {
-                cppFlags += "-std=c++17"
-            }
-        }
 
         ndk {
             //noinspection ChromeOsAbiSupport
-            abiFilters += listOf("arm64-v8a", "x86_64")
+            abiFilters += listOf("arm64-v8a")
```

Comment:
Yes, x86_64 is needed for the emulator since almost all the available virtual device API's, including the default API 35 uses the x86_64 ABI. However, while this support is nice to include, the app rarely runs on the emulator usually crashing immediately. The few times I was able to get it to work, it produces responses very slowly (1 token every second or two). Don't know if there's anything that can be done on our end about that or not.